### ISSUE TITLE
Icon not in main image

### ISSUE THREAD

**Issue by amsinha9** [rordenlab/dcm2niix#514]
I am trying to convert from DICOM to nifti from the ADNI dataset, but am running into several issues with the conversion. Here is the command I'm running:

```
dcm2niix_afni -o /output_dir/ -z y /input_dir-to-subfolder with all DICOM files for respective subject/
```

but keep running into this error:

Warning: Assuming 7FE0,0010 refers to an icon not the main image
Warning: Assuming 7FE0,0010 refers to an icon not the main image
slices stacked despite varying acquisition numbers (if this is not desired recompile with 'mySegmentByAcq')
Warning: Weird CSA 'ProtocolSliceNumber' (System/Miscellaneous/ImageNumbering reversed): VALIDATE SLICETIMING AND BVECS

It still somehow converts the DICOMs to a nifti file, but when I look at it in AFNI, it doesn't look quite right. Could you provide some help with this?

**Comment by neurolabusc** [comment]
- It is hard to provide help when you do not complete the template for a new issue. Specifically please report the complete version string, e.g. `dcm2niiX version v1.0.20201207  Clang12.0.0 ARM (64-bit MacOS)`, and make sure you are using the [latest stable release](https://github.com/rordenlab/dcm2niix/releases).

- As noted previously, not all [ADNI DICOM volumes are valid](https://github.com/rordenlab/dcm2niix/issues/416). This is a limitation with the ADNI dataset, not dcm2niix.

**Comment by amsinha9** [comment]
I apologize for not including the version string in the original post. The version string is:

```
Chris Rorden's dcm2niiX version v1.0.20181125  GCC4.3.2 (64-bit Linux)
```
I believe this could be due to not using the latest stable release, correct?

**Comment by neurolabusc** [comment]
Latest [stable release](https://github.com/rordenlab/dcm2niix/releases) is v1.0.20210317, so your version is 2.5 years out of date.

**Comment by neurolabusc** [comment]
dcm2niix discriminates between a `warning` (converted images, but anomalies were detected) and an `error` (unable to convert images). In your case, there is a `warning` which means that the conversion completed but dcm2niix is not confident that the conversion was successful. You will have to use visual inspection to ensure correct conversion. 

dcm2niix is designed to read the complicated DICOM format. I suspect these images were corrupted by a [GE GEIIS](https://github.com/rordenlab/dcm2niix/issues/367) PACS which inserted a thumbnail image in a manner that is not consistent with the DICOM standard. Some ADNI images I have seen only remove part of this thumbnail. I have tried to add kludges to detect these defects and convert the images correctly. However, I have only examined a small number of images, and I am not sure if my kludge is 100% successful. In brief, this is an issue with the images, not my software. A better solution would be for you to contact the ADNI folks and ask them to clean up their images.
